src/Tribe/Field.php

```diff
@@ -611,22 +611,77 @@ public function dropdown_select2() {
 		 * @return string the field
 		 */
 		public function license_key() {
-			$field = $this->doFieldStart();
-			$field .= $this->doFieldLabel();
-			$field .= $this->doFieldDivStart();
+			$field = $this->do_field_start();
+			$field .= $this->do_screen_reader_label();
+			$field .= $this->do_field_div_start();
 			$field .= '<input';
 			$field .= ' type="text"';
-			$field .= $this->doFieldName();
-			$field .= $this->doFieldValue();
+			$field .= $this->do_field_name();
+			$field .= $this->do_field_value();
 			$field .= '/>';
 			$field .= '<p class="license-test-results"><img src="' . esc_url( admin_url( 'images/wpspin_light.gif' ) ) . '" class="ajax-loading-license" alt="Loading" style="display: none"/>';
 			$field .= '<span class="key-validity"></span>';
-			$field .= $this->doScreenReaderLabel();
-			$field .= $this->doFieldDivEnd();
-			$field .= $this->doFieldEnd();
+			$field .= $this->do_screen_reader_label();
+			$field .= $this->do_field_div_end();
+			$field .= $this->do_field_end();
 
 			return $field;
 		}
 
+		/* deprecated camelCase methods */
+		public function doField() {
+			_deprecated_function( __METHOD__, '4.3', 'do_field' );
```

Comment:
Let's reference the class name in addition to the method name in the 3rd parameter:

```
_deprecated_function( __METHOD__, '4.3', __CLASS__ . '::do_field' );
```